OKTA-554534 Updates to dev docs for event hook filtering (#4062)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/event-hooks/index.md

Event types include user lifecycle changes, the completion by a user of a specific stage in an Okta process flow, and changes in Okta objects. You can configure an event hook, for example, to deliver notifications of user deactivation events. You can use hooks to trigger processes that you need to execute internally every time a user is deactivated. For example, updating a record in an HR system, creating a ticket in a support system, or generating an email message.

<ApiLifecycle access="ea" />

> You can reduce the number of event hooks triggered by defining filters on specific instances of the subscribed event type. For example, if you want a hook triggered by user sign-in events for a specific group of users, you can filter on that group, rather than having an event hook trigger for every user sign in. See [Create an event hook filter](#create-an-event-hook-filter).

Reviewer: Would something like this still work?
```suggestion
You can reduce the number of event hooks by defining filters on specific instances of the subscribed event type. For example, you could create a single event hook that is triggered by sign-in events for a specific group of users, rather than individual hooks each user. See [Create an event hook filter](#create-an-event-hook-filter).
```

Author: No. But I was being careless with the terminology, so, good catch. My revision:

You can reduce the number of event hook calls by defining filters on specific instances of the subscribed event type. For example, if you want an event hook call triggered by user sign-in events for a specific group of users, you can filter on that group, rather than having an event hook call trigger for every user sign in.

For a working example of an end-to-end event hook setup, see the [Event hook guide](/docs/guides/event-hook-implementation).

> > **Note:** It may take up to a few minutes before events are sent to the event hook after it’s created or updated.

Reviewer: ```suggestion
> **Note:** It may take several minutes before events are sent to the event hook after it’s created or updated.
```
